third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -143,9 +143,6 @@ struct LoadStoreConversionBase {
       : targetInfo(targetInfo), axisAnalysisPass(axisAnalysisPass) {}
 
   unsigned getContiguity(Value ptr) const {
-    auto tensorTy = dyn_cast<RankedTensorType>(ptr.getType());
```

Comment:
`ModuleAxisInfoAnalysis::getContiguity` does this internally:
https://github.com/triton-lang/triton/blob/4327b5b62f2abab716f275e52f4c5535785a2ab7/lib/Analysis/AxisInfo.cpp#L1250-L1252